packages/database/src/migrations/20210106000258-HeatmapOfVillageConfirmedAndSuspectedCases-modifies-data.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+async function deleteMapOverlay(db, mapOverlayId) {
+  return db.runSql(`
+    DELETE FROM
+      "mapOverlay"
+    WHERE
+      "id" = '${mapOverlayId}';
+  `);
+}
+
+async function deleteMapOverlayInMapOverlayGroupRelation(db, childId) {
+  return db.runSql(`
+    DELETE FROM
+      "map_overlay_group_relation"
+    WHERE
+      "child_id" = '${childId}';
+  `);
+}
+
+const mapOverlay = {
+  id: 'COVID_WS_Home_Village_Of_Suspected_And_Confirmed_Cases',
+  name: 'Home village of suspected and confirmed cases',
+  userGroup: 'Donor',
+  dataElementCode: 'value',
+  measureBuilderConfig: {
+    dataValues: {
+      WS_COVID19_Clinsurv_11: '*',
+    },
```

Comment:
I have the impression that this is not required, since we don't need to filter based on any value, just count all responses submitted in a village. The card's requirements state

> Count of cases by WS_COVID19_Clinsurv_11 (Current village)

I believe that in practice this means "count all responses in a village (which is captured via `WS_COVID19_Clinsurv_11`)"

No need to change this though, just for your reference!

```suggestion
```

Reply:
Yes Kostas, you are right. Here the hierachy of entities is `case -> village -> facility -> country`, and we are counting all the `case` (which is `WS_COVID19_Clinsurv_11`) in a village, which are 'confirmed' and 'suspected' cases.